### DISCUSSION TITLE
How Scalable is the Scaler Plan?

### DISCUSSION THREAD

**Discussion by jericopulvera** [planetscale/discussion#387] [Q&A] [answered]
Is the PlanetScale Scaler plan suitable for our use case where we have a AWS Aurora V2 single instance that peaks at 80 ACU and receive 2000 read requests per minute with most of our endpoints being cached using Database driver? We want to migrate to PlanetScale to save cost and have optimized our queries to minimize row reads on large datasets. The PlanetScale documentation does not mention any limits for the Scaler plan and we understand it is shared with multiple users, but does not support sharding.

Reference:
https://planetscale.com/docs/concepts/deployment-options
https://planetscale.com/pricing

**Comment by johnyerhot** [comment, marked as answer]
Hi @jericopulvera – you're right in your understandings. I believe someone from Sales has or will be reaching out to get a bit more information from you so we can give you an honest estimate.  If you don't hear back let me know.

That said, it sounds like you might be at or slightly above what our Scaler plan offers but we have options that are not a straight jump to the Teams plan, which is quite a bit more.
